pluto-message-ingestion/scripts/dynamo.js

```diff
@@ -0,0 +1,29 @@
+import { DynamoDBClient } from '@aws-sdk/client-dynamodb';
+import { DynamoDBDocumentClient, GetCommand } from '@aws-sdk/lib-dynamodb';
+
+const client = new DynamoDBClient({});
+
+const docClient = DynamoDBDocumentClient.from(client);
+
+/**
+ *
+ * @param {string} tableName
+ * @param {string} id
+ * @return {Promise<Record<string, any> | undefined>}
+ */
+export async function getProject(tableName, id) {
```

Comment:
is this used? It looks like a duplicate of the function from aws.js which is the one imported into the main script file

Reply:
nope! deleted now, thanks!